invenio_administration/assets/semantic-ui/js/invenio_administration/theme.js

```diff
@@ -0,0 +1,57 @@
+import $ from "jquery";
+
+const menuId = "#invenio-admin-top-nav"
+
+/* Expand and collapse navbar  */
+const toggleIcon = $("#rdm-burger-menu-icon");
```

Comment:
I think we should refactor it and move to invenio-theme, but maybe in a separate PR

Reply:
yes i agree!